manywheel/build_docker.sh

```diff
@@ -76,21 +76,22 @@ case ${GPU_ARCH_TYPE} in
         MANY_LINUX_VERSION="aarch64"
         DOCKERFILE_SUFFIX="_cuda_aarch64"
         ;;
-    rocm)
+    rocm|rocm-manylinux_2_28)
```

Comment:
Better to keep this as a separate case, following the same pattern.

Reply:
Actually, that's what I had started out with, following the pattern for cuda in the file, but then I came across this nifty way to cover more than one case in bash switch statement. And given the minor differences between the two cases, I'd rather not repeat the common stuff in both, following the [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). Let me know if you still feel like consolidating the two has any drawbacks.